src/seedu/addressbook/AddressBook.java

```diff
@@ -95,8 +95,8 @@
     private static final String PERSON_DATA_PREFIX_PHONE = "p/";
     private static final String PERSON_DATA_PREFIX_EMAIL = "e/";
 
-    private static final String PERSON_STRING_REPRESENTATION = "%1$s " // name
-                                                            + PERSON_DATA_PREFIX_PHONE + "%2$s " // phone
+    private static final String PERSON_STRING_REPRESENTATION = "%1$s," // name
```

Comment:
While I understand what you are trying to do, but it is not exactly exact implementation of your intention. 

The underlying data is actually look like the input (`John Appleseed p/9999999 e/test@example.com`). If you are transforming the data to CSV format, it should look more like `John Appleseed,9999999,test@example.com`. 

What you have done is just replacing the space to comma, which doesn't strip away the excess prefix that are used to identify the information.